sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,374 @@
+- Defined backend macros based on compiler which allows to compile an application for a backend without having to install the matching runtime. intel/llvm#15012
+- Reduced potential false possitives from address sanitizer by properly cleaning up private shadow memory. intel/llvm#15065
+- Disabled attribute propagation specified by SYCL 1.2.1 and removed remaining SYCL 2017/1.2.1 compatibility elements, including `-Wsycl-strict` diagnostics related to SYCL 2017 compatibility. intel/llvm#14984
+- Ensured that compiler-generated integration header/footer are warning-free to avoid breaking `-Werror` builds, especially when 3rd-party host compiler is used. intel/llvm#15171 intel/llvm#15175 intel/llvm#15197
+- Added pass to finish SROA for cooperative matrices. intel/llvm#15038
+- Removed `-full-main-file-name` and `-fsycl-use-main-file-name` options because `-include-footer` option is sufficient. intel/llvm#15072
+- Added more supported aspects for CPU AOT target. intel/llvm#15263
+- Removed workaround for `readnone` function parameter attribute as it is properly handled by CPU runtime now. intel/llvm#15298
+- Added diagnostic for bad argument with `-fsycl-device-obj`. intel/llvm#15381
+- Added a warning for cases when kernel-only attributes are applied to non-kernel functions. intel/llvm#15154
+- Started obeying `-nocudalib` when linking against libdevice. intel/llvm#15378
+- Fixed misleading diagnostic about non-external function/variable when applying attributes like  `[[sycl_device]]` or `[[intel::device_indirectly_callable]]` on  functions/variables without external linkage. intel/llvm#15372
+- Added `--only-needed` flag to llvm-link for AMDGCN. intel/llvm#15558
+- Native CPU now uses a pass to convert ABI-compliant calls to mux builtins, removing vector versions and ensuring the interface remains unaffected by the ABI. intel/llvm#15592
+- The `-fsycl-link=image` behavior is updated to package host objects similarly to `-fsycl-link=early`, ensuring proper linking, especially on Windows. intel/llvm#15539
+- Enabled generation of approximate `div`/`sqrt` with `-ffast-math` for NVTPX. intel/llvm#15553
+- Added extra optimization passes in Native CPU pipeline. intel/llvm#14380
+- Updated implementation of `-fsycl-host-compiler` option to only use user-provided hints (i.e. `PATH`) to locate the specified compiler. Previously this option also took into account a few extra locations implicitly known to the toolchain, potentially causing incorrect binary to be used. intel/llvm#15769
+- Deprecated `[[intel::reqd_sub_group_size]]`, the official SYCL 2020 spelling should be used instead (with `sycl::` namespace). intel/llvm#15798
+- Disabled ITT annotations in device code by default to reduce device code size. intel/llvm#14910
+- Enabled floating point atomics via `atomicrmw` instructions for Native CPU. intel/llvm#15888
+- Enabled nonsemantic debug info by default (except fpga target). This should slightly improve debugging experience. intel/llvm#16120
```

Comment:
```suggestion
- Enabled nonsemantic debug info by default (except fpga target) improving debugging experience for device code. intel/llvm#16120
```
Q: is FPGA still a part of this release?

Reply:
No, thanks, removed `(except fpga target)`